lib/sensu/settings/loader.rb

```diff
@@ -124,7 +124,7 @@ def load_file(file)
       def load_directory(directory)
         warning("loading config files from directory", :directory => directory)
         path = directory.gsub(/\\(?=\S)/, "/")
-        Dir.glob(File.join(path, "**/*.json")).each do |file|
+        Dir.glob(File.join(path, "**{,/*/**}/*.json")).uniq.each do |file|
```

Comment:
The glob seems to match files multiple times:

```
[7] pry(main)> Dir.glob(File.join("**/*.json")) { |i| puts i }
nested/file.json
merger.json
=> nil
[8] pry(main)> Dir.glob(File.join("**{,/*/**}/*.json")) { |i| puts i }
nested/file.json
merger.json
nested/file.json
alternative/symlinked.json
=> nil
```

Took the glob from here: http://stackoverflow.com/questions/357754/can-i-traverse-symlinked-directories-in-ruby-with-a-glob